src/documentation/generate.js

```diff
@@ -71,6 +72,13 @@ function pruneFolder(schemaPruner, folderpath) {
             let query = fs.readFileSync(path.join(folderpath, file), 'UTF-8');
             schemaPruner.process(query);
         });
+
+    files
```

Comment:
Please add a test for this in `SchemaPrunerTest.js` ... you diff coverage is ... 0% ;-)

Reply:
`generate.js` is ignored by unit tests.